src/bin/cargo/cli.rs

```diff
@@ -279,6 +279,9 @@ fn expand_aliases(
     args: ArgMatches,
     mut already_expanded: Vec<String>,
 ) -> Result<(ArgMatches, GlobalArgs), CliError> {
+    println!("gctx == {:#?}", gctx);
```

Comment:
Could you remove intermediate commits during development? They are not necessary for either review or better understanding in the future.

Reply:
The `println!` still exists in the commit history. Please remove it.